src/main/java/at/hannibal2/skyhanni/test/SkyHanniConfigSearchResetCommand.kt

```diff
@@ -107,13 +114,46 @@ object SkyHanniConfigSearchResetCommand {
         val shimmy = Shimmy.makeShimmy(root, list) ?: return "§cCould not change config element '$term', not found!"
         return try {
             shimmy.setJson(element)
-            "§eChanged config element $term."
+            "§eChanged config element $term to $rawJson."
         } catch (e: Exception) {
-            ErrorManager.logError(e, "Could not change config element '$term' to '$rawJson'")
+            ErrorManager.logErrorWithData(
+                e, "Could not change config element '$term' to '$rawJson'",
+                "args" to args.toList()
+            )
             "§cCould not change config element '$term' to '$rawJson'"
         }
     }
 
+    private suspend fun toggleCommand(args: Array<String>): String {
+        if (args.size == 1 || args.size == 3) return "§c/shconfig toggle <config name> [value 1] [value 2]"
+
+        val path = args[1]
+        val rawJson1 = if (args.size > 2) args[2] else "true"
+        val rawJson2 = if (args.size > 2) args[3] else "false"
+
+        return try {
+            val (argsFilter) = createFilter(true) { path.lowercase() }
+            val (classFilter) = createFilter(false) { path.lowercase() }
+
+            val currentValue = findConfigElements(argsFilter, classFilter, true).toString()
+            val newValue = when (currentValue) {
+                "[$rawJson1]" -> rawJson2
+                else -> rawJson1
+            }
+            setCommand(arrayOf("set", path, newValue))
+
+        } catch (e: Exception) {
+            ErrorManager.logErrorWithData(
+                e, "Error while trying to toggle config element",
+                "args" to args.toList(),
+                "path" to path,
+                "value1" to rawJson1,
+                "value2" to rawJson2,
```

Comment:
these three are already included in args.toList()

Reply:
well value1 and value2 arent exactly but ig no point